app/src/main/res/values/fonts.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+  <string name="default_typeface_regular">fonts/Quicksand-Regular.ttf</string>
```

Comment:
Why not using attributes? This is not really what I had outlined... I meant:

```xml
In attrs.xml:
<attr name="titleTypeface" format="string" />
<attr name="defaultRegularTypeface" format="string" />
<attr name="defaultMediumTypeface" format="string" />
<attr name="defaultBoldTypeface" format="string" />
<attr name="defaultItalicTypeface" format="string" />

In themes.xml:
<style name="Theme.Squanchy">
  <item name="titleTypeface">fonts/leaguespartan-bold.otf</item>
  <item name="defaultRegularTypeface">fonts/Quicksand-Regular.ttf</item>
  <item name="defaultMediumTypeface">fonts/Quicksand-Medium.ttf</item>
  <item name="defaultBoldTypeface">fonts/Quicksand-Bold.ttf</item>
  <item name="defaultItalicTypeface">fonts/Quicksand-Italic.ttf</item>
</style>

In styles.xml:
<style name="Whatever.Title">
  <item name="fontPath">?attr:titleTypeface</item>
</style>
```

Would that not work?